src/all/kamyroll/src/eu/kanade/tachiyomi/animeextension/all/kamyroll/Yomiroll.kt

```diff
@@ -65,6 +71,10 @@ class Yomiroll : ConfigurableAnimeSource, AnimeHttpSource() {
         super.client.newBuilder().addInterceptor(tokenInterceptor).build()
     }
 
+    private val noTokenClient by lazy {
+        super.client.newBuilder().build()
+    }
```

Comment:
This is a nitpick but you don't have to build a new client, you can just do

```suggestion
    private val noTokenClient = super.client
```

Reply:
Isn't the default client the cloudflare one? That was my thought But I guess it doesn't matter here. Thanks